frontend/src/types/ClassroomTypes.tsx

```diff
@@ -5,7 +5,7 @@ export type ClassroomInput = "className" | "startDate" | "gradeLevel";
 export interface ClassroomForm {
   className: string;
   startDate: Date | undefined;
-  gradeLevel: Grade;
+  gradeLevel?: Grade;
```

Comment:
why optional?

Reply:
I was getting some data that had missing grade levels. Practically it won't happen in prod but I figured it didn't hurt to make the code on the frontend handle the extra case if it can occur possibly